lib/howl/variables/core_variables.moon

```diff
@@ -45,9 +52,9 @@ config.define {
   description: 'When to automatically inspect code for abberrations'
   default: 'idle'
   options: {
-    { 'manual', 'Only inspect when explicitly asked' }
-    { 'idle', 'Inspect on idle' }
-    { 'save', 'Inspect when saving a buffer' }
+    { 'manual', 'Run all inspectors when explicitly asked' }
+    { 'idle', 'Run on-idle inspectors on idle and on-save inspectors on save' }
+    { 'save', 'Run all inspectors, but only on save' }
```

Comment:
I feel these three options are useful but names can be confusing? E.g. 'idle' runs save inspectors too (and vice-versa). Maybe 'off' for manual (*auto*_inspect is off implies manual), 'on' for the default setting and 'save' (can't think of a better name here).